src/healthcheck.ts

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env node
 import { HOST, RPC_PORT } from './constants.js'
-import { logger } from './logger.js'
-
-const log = logger.forComponent('healthcheck')
 
 /**
- * This healthcheck script is used to check if the server is running and healthy.
+ * This healthcheck script is used to check if the server is running and healthy
  */
-const rootReq = await fetch(`http://${HOST}:${RPC_PORT}/api/v0/http-gateway-healthcheck`, { method: 'GET' })
+const rootReq = await fetch(`http://${HOST}:${RPC_PORT}/api/v0/http-gateway-healthcheck`, {
+  method: 'GET'
+})
 const status = rootReq.status
```

Comment:
nit: are we getting any logging information now that we're removing the log here?

the prior log was probably too verbose (log every 12 seconds..) but I just want to make sure we can see some type of healthcheck item if we need to